src/controllers/applicationController.ts

```diff
@@ -142,4 +142,42 @@ export const getAllApplications = async (req: Request, res: Response) => {
     const appRepo = getManager().getRepository(Application);
     const applications = await appRepo.find();
     res.status(200).send(applications);
+}
+
+export const modifyResume = async (req: Request, res: Response) => {
+    const appRepo = getManager().getRepository(Application);
+    const token = req.header('Authorization')?.split(' ')[1];
+    if(!token) {
+        res.sendStatus(401);
+        return;
+    }
+
+    const user = await auth.getUserObjectFromToken(token, ['application']);
+    if(!user) {
+        res.sendStatus(401);
+        return;
+    }
+
+    if(!user.application) {
+        res.status(400).send('User does not have an application yet.');
+        return;
+    } 
+
+    // Handle the resume. We're saving them to a resumes folder in the root of the project
+    if(req.files?.resume) {        
+        let resume = req.files.resume;
+        try {
+            const resumePath = await fs.promises.realpath(RESUME_ROOT) + '/' + user.application.email + '.pdf';
+            await resume.mv(resumePath);
```

Comment:
This will overwrite the existing file, correct?

Reply:
Yup